src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+.stars__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
Avoid using tag names for styling. It's a good practice to style elements using classes to ensure that your styles are not overly broad and potentially conflicting with other styles. Consider adding a class to the `.stars_star` elements and style them accordingly.